shared-kcp/sharedkcp.kubeconfig

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
```

Comment:
This should also probably be in its own folder.

Reply:
Done.